### PR TITLE
add channel 'email' for trigger form

### PR DESCRIPTION
**Pull request by GarageInc**
**Summary** 

Enable channel 'email' in signal form

**Screenshots**
image

### CODE REVIEW

File: src/ducks/Signals/signalFormManager/signalCrudForm/formParts/channels/TriggerFormChannels.js [hunk 1]
```diff
@@ -139,6 +140,15 @@ const TriggerFormChannels = ({
           <FormikLabel text='Notify me via' />
         </SidecarExplanationTooltip>
         <div className={styles.notifyBlock}>
+          <ChannelCheckbox
+            channel={CHANNEL_NAMES.Email}
+            isActive={isActive}
+            isDisabled={isDisabled}
+            toggleChannel={toggleChannel}
+            isRequired={isRequired}
+            recheckBrowserNotifications={recheckBrowserNotifications}
```
Comment (Partysun) [inline src/ducks/Signals/signalFormManager/signalCrudForm/formParts/channels/TriggerFormChannels.js]: Why do we need to do this for EMAIL channel? @GarageInc

File: src/ducks/Signals/signalFormManager/signalCrudForm/formParts/channels/TriggerFormChannels.js [hunk 2]
```diff
@@ -139,6 +140,15 @@ const TriggerFormChannels = ({
           <FormikLabel text='Notify me via' />
         </SidecarExplanationTooltip>
         <div className={styles.notifyBlock}>
+          <ChannelCheckbox
```
Comment (Partysun) [inline src/ducks/Signals/signalFormManager/signalCrudForm/formParts/channels/TriggerFormChannels.js]: Can we go through array of channels here?

File: src/ducks/Signals/signalFormManager/signalCrudForm/formParts/channels/TriggerFormChannels.js [hunk 3]
```diff
@@ -11,7 +11,7 @@ import SidecarExplanationTooltip from '../../../../../SANCharts/SidecarExplanati
 import { CHANNEL_NAMES } from '../../../../utils/constants'
 import styles from '../../signal/TriggerForm.module.scss'
 
-const permanentDisabledChannels = [CHANNEL_NAMES.Email]
+const permanentDisabledChannels = []
```
Comment (Partysun) [inline src/ducks/Signals/signalFormManager/signalCrudForm/formParts/channels/TriggerFormChannels.js]: We can remove this. It's unused anymore

File: src/ducks/Signals/signalFormManager/signalCrudForm/formParts/channels/TriggerFormChannels.js (hunk 2 above)
Comment (GarageInc) [reply src/ducks/Signals/signalFormManager/signalCrudForm/formParts/channels/TriggerFormChannels.js]: We can

File: src/ducks/Signals/signalFormManager/signalCrudForm/formParts/channels/TriggerFormChannels.js (hunk 3 above)
Comment (GarageInc) [reply src/ducks/Signals/signalFormManager/signalCrudForm/formParts/channels/TriggerFormChannels.js]: Can be removed

File: src/ducks/Signals/signalFormManager/signalCrudForm/formParts/channels/TriggerFormChannels.js (hunk 1 above)
Comment (GarageInc) [reply src/ducks/Signals/signalFormManager/signalCrudForm/formParts/channels/TriggerFormChannels.js]: Line 149? It is a wrapped component, which shows dialog with settings. Dialog not only with email, but also with push/telegram channels
image

**Pull request merged**